sdk/core/System.BinaryData/tests/BinaryDataTests.cs

```diff
@@ -63,10 +49,7 @@ public void ToStringRespectsArraySegmentBoundaries()
             var payload = "pre payload post";
             var bytes = Encoding.UTF8.GetBytes(payload);
```

Comment:
Where possible, please use dotnet/runtime style rules in these tests  (`var` is only permitted with `new`, `as`-casting, and "hard" casting).

It'll make migration easier.